lib/intercom_flutter.dart

```diff
@@ -117,6 +143,20 @@ class Intercom {
     return _channel.invokeMethod('sendTokenToIntercom', {'token': token});
   }
 
+  static Future<dynamic> registerIosTokenToIntercom() {
+    print("[INTERCOM_FLUTTER] Start sending iOS token to Intercom");
+    if (_iosDeviceToken != null) {
+      return _channel
+          .invokeMethod('sendTokenToIntercom', {'token': _iosDeviceToken});
+    } else {
+      return throw ErrorDescription("[INTERCOM_FLUTTER] No iOS Device token");
```

Comment:
```suggestion
      return throw ErrorDescription("No iOS device token was generated.");
```

Also, add some info on how to solve this issue (e.g. calling the method to generate it)